{crossTime: 0.3, errorAng: 3, errorCorrectionDis: 600, errorMoibleAng: 3.0,
  errorMoibledis: 1000, errorRotate0: 8, errorRotate90: 83, maxVel: 10.0, minVel: 25,
  rotateSlowAng: 20, rotateYaw: 45, velYaw: 25}
